### DISCUSSION TITLE
demo error in 0301

### DISCUSSION THREAD

**Discussion by mikatakahashi** [IBAHCM/RPiR#223] [Archive] [answered]
Hi @soniamitchell 
I have created function file and saved in demo and devtooled for document and install. and I get this error message when I try to check if my demo works.

> demo("d0302_stochastic_birth_death.R", package = "MikaTakahashiSeries03")
Error in demo("d0302_stochastic_birth_death.R", package = "MikaTakahashiSeries03") : 
  No demo found for topic ‘d0302_stochastic_birth_death.R’

**Comment by DanniAnderson** [comment]
I get the exact same thing.  Its in my demo file, 00Index and the list when I do " demo(package = "DavidAndersonSeries03") " but not the next step.

I also get an error when I type " ?timestep_stochastic_birth_death "

> No documentation for ‘timestep_stochastic_birth_death’ in specified packages and libraries:
> you could try ‘??timestep_stochastic_birth_death’

Even though I have that in my R folder.

**Reply by soniamitchell** [reply]
@DavidAnderson102 yours works fine for me. Try

```
library(devtools)

# Build documentation
document() 

# Install package (maybe you didn't install since making changes)
install()
```

**Reply by DanniAnderson** [reply]
Hi Sonia,

I've got it working now.  Just noticed in exercise 3-1 in the Check it works section it says use...

> Try running your demo with  demo("d0302_stochastic_birth_death.R", package = "githubusernameSeries03")

... but it doesn't work because of the ".R" as it works then you take it out.

Thanks

**Reply by soniamitchell** [reply]
Well spotted! I've fixed it now 👍

**Comment by soniamitchell** [comment, marked as answer]
Hey @mikatakahashi, try 

```
demo("d0302_stochastic_birth_death”, package = "MikaTakahashiSeries03")
```

The first argument requires the demo name, without the file extension. Hope that helps!

**Reply by mikatakahashi** [reply]
Hi @soniamitchell 
Thank you, i tried it and it returned with another error saying;
Error: unexpected symbol in "demo("d0302_stochastic_birth_death”, package = "MikaTakahashiSeries03"

Do you have any idea what the unexpected symbol is in this?

**Reply by soniamitchell** [reply]
Hey @mikatakahashi, can you take a screen shot? From the error you pasted I'm not sure if there's an extra quote sign at the start, or a missing bracket at the end, or something else in the error message I can't see..

**Reply by mikatakahashi** [reply]
Yes!
<img width="724" alt="Screen Shot 2021-02-03 at 20 03 22" src="https://user-images.githubusercontent.com/77616349/106802435-dfa2a480-665a-11eb-9993-c2349225999f.png">
<img width="1426" alt="Screen Shot 2021-02-03 at 20 04 19" src="https://user-images.githubusercontent.com/77616349/106802546-0234bd80-665b-11eb-8c1f-e1e32032da23.png">

I hope this works. Thank you!

**Reply by soniamitchell** [reply]
Ahh.. did you copy and paste what I wrote? Sometimes that causes quote signs to go weird.. note the curly quote sign after `death`. Try deleting the quote and replacing it with another quote sign 👍

**Reply by mikatakahashi** [reply]
Wow... ok that was surprising. Thank you for spotting it. It's working now!

**Reply by soniamitchell** [reply]
No worries 👍
